monkestation/code/modules/hydroponics/seeds/coconut.dm

```diff
@@ -0,0 +1,33 @@
+/obj/item/seeds/coconut
+	name = "coconut tree"
+	desc = "A coconut tree"
+	seed_offset = -12
+	icon = 'monkestation/icons/obj/hydroponics/fruit.dmi'
```

Comment:
just a sanity check to make sure this isn't a goon sprite considering line 8 of this file

Reply:
yea the seeds and the coconut fruit itself were done by me the tree is goon